app/src/main/res/layout/earn_tab.xml

```diff
@@ -24,16 +24,16 @@
 
         <TextView
             android:id="@+id/balance_text"
```

Comment:
need to disable clicks on the button so no double clicks

Reply:
done